radis/lbl/factory.py

```diff
@@ -739,21 +740,15 @@ def eq_spectrum(
         conditions = self.get_conditions()
         conditions.update(
             {
-                "calculation_time": self.profiler.dict_time["spectrum_calc_before_obj"][
-                    0
+                "calculation_time": self.profiler.final[list(self.profiler.final)[-1]][
+                    "spectrum_calc_before_obj"
```

Comment:
@erwanp  since `spectrum_calc_before_obj` sums up all steps, we should remove this and maybe add a sum function which will add time taken per verbose level; Calculation time should be equal to the whole time taken by spectrum till generating and saving the spectrum?